### PR TITLE
Milestone five

### PR DESCRIPTION
**Pull request by geraldgsh**
1. Implement the handleClick function in app.js
2. Add buttonName as an argument to handleClick function in app.js
3. Pass the total or next value to Display to display the current result in app.js
4. Pass handleClick as a clickHandler property to ButtonPanel in app.js
5. Implement the handleClick function in buttonPanel.js
6. Add buttonName as an argument which returns clickHandler from props in buttonPanel.js
7. Pass the clickHandler property to each button in buttonPanel.js
8. Implement the handleClick function in button.js
9. Add buttonName as an argument which returns clickHandler from props in button.js
10. Implement onClick event to attach the handleClick event to the button in button.js
11. Add display conditions in calculate.js
12. Stringify operands in operate.js

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  const [one, two] = [Big(numberOne), Big(numberTwo)];
+  switch (operation) {
+    case '+':
+      return one.plus(two).toString();
+    case '-':
+      return one.minus(two).toString();
+    case 'X':
+      return one.times(two).toString();
+    case '÷':
+      return one.div(two).toString();
```
Comment (shloch) [inline src/logic/operate.js]: manage division by ZERO error.........You can display an error on screen or something (`'INFNITY`)....just handle it
Screen Shot 2020-04-11 at 4 41 34 PM

**Review by shloch — CHANGES_REQUESTED** [review]
## perfect job @geraldgsh 

Screen Shot 2020-04-11 at 4 45 06 PM
### one more thing below, before thumbs up ......let's do this !!

**Comment by geraldgsh** [comment]
Thanks @shloch, i'll make amendments accordingly.

**Review by shloch — APPROVED** [review]
### we're good to go...............on to next project

accomplished

**Pull request merged**